### PR TITLE
fix #17: Upgrade vsc-engine to v1.74

### LINKED ISSUE #17: Update vscode-engine from v1.17 to v1.74
I'm having some trouble running the current implementation of the code, I think there are some deprecated requirements.

Can I upgrade the engine?

### PR DESCRIPTION
**Pull request by mariogarridopt**

### CODE REVIEW

**Comment by jvalecillos** [comment]
Hello, thanks for the contribution.

At first when I started developing the extension I intentionally avoided updating the engine to the latest version of VSCode since ~it is not clear/documented~ I needed to remember how the compatibility works around older versions of the IDE and the extensions. It has been a while since I updated the dependencies and I noticed there are a few changes regarding the tests packages.

I'm gonna check a few things before going ahead with updating the engine to make sure that updating all the dependencies at once it is not an issue.

I'll keep you posted.

**Comment by jvalecillos** [comment]
I updated my plan about the engine update in a comment #17
TL;DR: I'm updating the engine but to slightly older version to give more support for older versions of the IDE.

**Comment by jvalecillos** [comment]
For now, this has been implemented in #20
